### PR TITLE
updated the tag of open source

### PR DESCRIPTION
**Pull request by sakshi025**
fixed the issue by updating tag image in readme file
I will be thankful if u can merge this

### CODE REVIEW

**Comment by guyluz11** [comment]
As I said in the issue, it had already been assigned to someone else.

**Comment by sakshi025** [comment]
> As I said in the issue, it had already been assigned to someone else.

what if he not complete....i had pulled request and its working...if he won't be able to complete then can u merge that pull request

**Comment by guyluz11** [comment]
Yep sure, ping me if in two days the issue is not closed

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
-[![dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg)](https://github.com/CyBear-Jinni/security_bear/actions?query=workflow%3A%22Dart+CI%22) [![style: lint](https://img.shields.io/badge/lint-1.3.0-blue)](https://pub.dev/packages/lint) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0) [![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://en.wikipedia.org/wiki/Open_source)
+[![dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg)](https://github.com/CyBear-Jinni/security_bear/actions?query=workflow%3A%22Dart+CI%22) [![style: lint](https://img.shields.io/badge/lint-1.3.0-blue)](https://pub.dev/packages/lint) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badge/)
```
Comment (guyluz11) [inline README.md:6]: Please change image url to `https://badges.frapsoft.com/os/v1/open-source-200x33.png?v=103`.
The .svg does not have a background 

image

Comment (guyluz11) [inline README.md:6]: We prefer it to go to the link of open source and not the badge GitHub link
So return link to be `https://en.wikipedia.org/wiki/Open_source`

**Review by guyluz11 — COMMENTED** [review]
Please fix changes

**Comment by guyluz11** [comment]
And change your pr to dev

**Comment by guyluz11** [comment]
Your other pr got merged.

Will close this pr.

**Pull request closed without merging**